### DISCUSSION TITLE
Differential expression testing post-integration

### DISCUSSION THREAD

**Discussion by andrewwbutler** [satijalab/seurat#4000] [Q&A] [answered]
After integration, which Assay should I use for differential expression testing?

**Comment by satijalab** [comment, marked as answer]
We recommend running your differential expression tests on the “original / unintegrated” data. By default this is stored in the “RNA” Assay. The integration procedure inherently introduces dependencies between data points. This violates the assumptions of the statistical tests used for differential expression. If you are concerned about additional confounders in the data such as batch or condition, these can be supplied to certain differential expression tests such as the logistic regression test (`test.use = “LR”` in `FindMarkers`) via the `latent.vars` parameter. We recommend this for each of the integration workflows (i.e. RPCA, sctransform, reference-based) implemented in Seurat.

**Reply by sam-212** [reply]
Hello, thanks for posting this question and the answer.
I'd like to ask something else:

Is there a method which can get a differential gene expression between integrated data sets, when they have been obtained from different platforms? I basically have integrated my data with published data. It's been useful in visualizing where the published data fits with mine using UMAP, but I'd really like more info on the gene expression differences. The difference in read depth between the two samples is significant. Would the LR test work in this situation?

**Reply by royfrancis** [reply]
Which slot is to be used within the RNA assay? I would guess deseq2 for example would like raw counts. Does MAST, wilcox etc prefer raw "counts" or normalized "data"? And do both these slots require the latent variables to be specified? Because I suppose the vars.to.regress only corrects scaled data and not normalised data?

**Reply by hyjforesight** [reply]
i have the same question as sam-212

**Reply by ZedekiahZhou** [reply]
Hello satijalab, thanks for you answers!
I am a little puzzled about "We recommend this for each of the integration workflows (i.e. RPCA, sctransform, reference-based) implemented in Seurat."   I understand what uou clarify that 'integrated' assay should not be used for differential expression tests, but for integration workflows on sctransformed data, should I performed differential gene expression on "RNA" assay or "SCT" assay?
Thanks!

**Reply by jiangshan529** [reply]
> We recommend running your differential expression tests on the “original / unintegrated” data. By default this is stored in the “RNA” Assay. The integration procedure inherently introduces dependencies between data points. This violates the assumptions of the statistical tests used for differential expression. If you are concerned about additional confounders in the data such as batch or condition, these can be supplied to certain differential expression tests such as the logistic regression test (`test.use = “LR”` in `FindMarkers`) via the `latent.vars` parameter. We recommend this for each of the integration workflows (i.e. RPCA, sctransform, reference-based) implemented in Seurat.

I have the same question with sam-212. Can someone answer this? Thanks!

**Reply by TeodoraTockovska** [reply]
> Hello satijalab, thanks for you answers! I am a little puzzled about "We recommend this for each of the integration workflows (i.e. RPCA, sctransform, reference-based) implemented in Seurat." I understand what uou clarify that 'integrated' assay should not be used for differential expression tests, but for integration workflows on sctransformed data, should I performed differential gene expression on "RNA" assay or "SCT" assay? Thanks!

A little late to your reply but I recommend performing the differential analysis on the "RNA" assay.
